jupyter-book/_toc.yml

```diff
@@ -60,14 +60,11 @@ parts:
 - caption: Selecting the best model
   chapters:
   - file: overfit/overfit_module_intro
-  - file: overfit/overfit_overfitting_underfitting_index
-    sections:
-    - file: overfit/overfitting_vs_under_fitting_slides
-  - file: overfit/overfit_validation_learning_curves_index
-    sections:
-    - file: overfit/learning_validation_curves_slides
-    - file: python_scripts/cross_validation_validation_curve
-    - file: python_scripts/cross_validation_learning_curve
+  - file: overfit/overfitting_vs_under_fitting_slides
+  - file: overfit/overfitting_vs_under_fitting_quiz_m2_01.md
+  - file: overfit/learning_validation_curves_slides
+  - file: python_scripts/cross_validation_ex_01.py
+  - file: python_scripts/cross_validation_sol_01.py
```

Comment:
Just checking: I see that the `overfit_` kind of files have been deleted, but I do not see the new ones have been added.

Reply:
Yes, these are `index` files and where there because the original material had separate sections for 'overfitting and underfitting' and 'learning curves'. I removed one level in the toc tree and put everything at the chapter level